gem/earlgrey.gemspec

```diff
@@ -0,0 +1,43 @@
+#
+#  Copyright 2016 Google Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+require_relative 'lib/earlgrey/version'
+
+Gem::Specification.new do |s|
+  s.required_ruby_version = '>= 2'
+
+  s.name          = 'earlgrey'
+  s.version       = EarlGrey::VERSION
+  s.license       = 'Apache-2.0'
+  s.summary       = 'EarlGrey installer gem'
+  s.description   = 'Command line tool for installing EarlGrey into an iOS Unit Testing target'
+  s.authors       = s.email = ['code@bootstraponline.com']
```

Comment:
Sure, I pushed a new update. Authors looks like this now:

<img width="254" alt="earlgrey_authors_gem" src="https://cloud.githubusercontent.com/assets/1173057/15275921/7b80d562-1aa6-11e6-8276-5bf59e1cad6a.png">